app/admin/admin_users.rb

```diff
@@ -154,6 +154,10 @@
   controller do
     skip_before_action :authenticate_active_admin_user, raise: false, only: :omg_wtf_omg
 
+    def scoped_collection
+      AdminUser.active
```

Comment:
A menos que estemos 100% seguros de que esto es lo que necesitamos, lo recomendado para overloadear este método es aplicar el scope sobre la llamada a `super`.
```suggestion
      super.active
```